src/plays/dev-jokes/DevJokes.tsx

```diff
@@ -1,11 +1,22 @@
 import PlayHeader from "common/playlists/PlayHeader";
 import "./devJokes.css";
 import axios from "axios";
-import { useEffect, useState } from "react";
+import {
+  useEffect,
+  useState,
+} from "react";
+import React from "react";
 import Spinner from "./Spinner";
-function DevJokes(props) {
-  const [joke, setjoke] = useState([]);
-  const [spinner, setspinner] = useState(false);
+
+interface jokes{
```

Comment:
```suggestion
interface Jokes{
```
@yung-coder , Interface name should start with a capital.